config/rbac/role.yaml

```diff
@@ -19,17 +19,15 @@ rules:
   - update
   - watch
 - apiGroups:
-  - e2etest.grpc.io
+  - ""
   resources:
-  - loadtests/status
+  - pods/status
   verbs:
   - get
```

Comment:
I totally agree that we should limit the privilege of each role to its minimum requirement, I am just wondering PodStatus is system generated information, we probably have no control over it except get, why did we set patch and update action for a status object?

Reply:
We didn't. The `pods/status` resources only declares the `get` verb.

Reply:
We have `get`, `patch` and `update` verbs on the `loadtests/status` resource.